Description: upstream version-script map
 Import a version-script map created with the help of cproto
 from the C header `def.h' and `macro.h'; meant to be submitted
 to the upstream maintainer.
Origin: debian
Author: Jerome Benoit <calculus@rezozer.net>
Last-Update: 2014-09-13

--- /dev/null
+++ b/symmetrica.map
@@ -0,0 +1,3097 @@
+LIBSYMMETRICA_2.0.0 {
+global:
+binom_values;
+no_mem_check;
+sym_no_results;
+cons_eins;
+cons_negeins;
+cons_null;
+cons_zwei;
+cons_drei;
+texout;
+check_time_co;
+zeilenposition;
+row_length;
+tex_row_length;
+integer_format;
+english_tableau;
+no_banner;
+kuerzen_yn;
+texmath_yn;
+scanoutput_yn;
+texposition;
+doffset;
+sym_background;
+sym_www;
+sym_timelimit;
+freeall_speicherposition;
+freeall_speichersize;
+freeall_speicher;
+SYM_free;
+SYM_malloc;
+SYM_realloc;
+SYM_calloc;
+freeall_speichersize_max;
+absolute;
+absolute_bruch;
+absolute_integervector;
+absolute_longint;
+absolute_matrix;
+absolute_vector;
+a_charvalue_co;
+a_charvalue;
+add;
+add_adjacency_matrix;
+add_apply;
+add_apply_bruch;
+add_apply_bruch_bruch;
+add_apply_bruch_bruch_pre261101;
+add_apply_bruch_integer;
+add_apply_bruch_scalar;
+add_apply_cyclo;
+add_apply_default;
+add_apply_ff;
+add_apply_gral;
+add_apply_gral_gral;
+add_apply_hashtable;
+add_apply_integer;
+add_apply_integer_bruch;
+add_apply_integer_integer;
+add_apply_integer_longint;
+add_apply_integervector;
+add_apply_laurent;
+add_apply_longint;
+add_apply_longint_integer;
+add_apply_longint_longint;
+add_apply_matrix;
+add_apply_matrix_matrix;
+add_apply_monopoly;
+add_apply_polynom;
+add_apply_polynom_polynom;
+add_apply_polynom_scalar;
+add_apply_polynom_schubert;
+add_apply_reihe;
+add_apply_scalar_bruch;
+add_apply_scalar_polynom;
+add_apply_schubert;
+add_apply_schubert_schubert;
+add_apply_sqrad;
+add_apply_symchar;
+add_apply_symfunc;
+add_apply_symfunc_symfunc;
+add_apply_vector;
+add_bruch;
+add_bruch_bruch;
+add_bruch_integer;
+add_bruch_scalar;
+add_cyclo;
+add_cyclo_cyclo;
+add_elmsym;
+add_elmsym_elmsym;
+add_ff;
+add_galois;
+add_homsym;
+add_homsym_homsym;
+add_integer;
+add_integer_integer;
+add_integer_longint;
+add_integervector;
+addinvers_apply;
+addinvers_apply_bruch;
+addinvers_apply_cyclo;
+addinvers_apply_elmsym;
+addinvers_apply_ff;
+addinvers_apply_galois;
+addinvers_apply_hashtable;
+addinvers_apply_homsym;
+addinvers_apply_integer;
+addinvers_apply_laurent;
+addinvers_apply_longint;
+addinvers_apply_monom;
+addinvers_apply_monomial;
+addinvers_apply_monopoly;
+addinvers_apply_polynom;
+addinvers_apply_powsym;
+addinvers_apply_schur;
+addinvers_apply_sqrad;
+addinvers_apply_symchar;
+addinvers_apply_vector;
+addinvers;
+addinvers_bruch;
+addinvers_cyclo;
+addinvers_ff;
+addinvers_integer;
+addinvers_longint;
+addinvers_matrix;
+addinvers_monom;
+addinvers_monopoly;
+addinvers_polynom;
+addinvers_reihe;
+addinvers_sqrad;
+addinvers_symchar;
+addinvers_vector;
+add_koeff;
+add_laurent;
+add_longint;
+add_longint_integer;
+add_longint_longint;
+add_matrix;
+add_matrix_matrix;
+add_monom;
+add_monomial;
+add_monomial_monomial;
+add_monom_schur;
+add_monopoly;
+add_monopoly_monopoly;
+add_partition;
+add_part_part;
+add_polynom;
+add_polynom_polynom;
+add_powsym;
+add_powsym_powsym;
+add_reihe;
+add_scalar_cyclo;
+add_scalar_monopoly;
+add_scalar_polynom;
+add_scalar_sqrad;
+add_schubert;
+add_schubert_schubert;
+add_schur;
+add_schur_schur;
+add_sqrad;
+add_sqrad_sqrad;
+add_staircase_part;
+add_symchar;
+addtoallvectorelements;
+add_vector;
+ak_make_alt_classes;
+ak_make_alt_partitions;
+ak_plet_phm_integer_partition_;
+all_01_matrices;
+allclasses;
+allclasssums;
+all_codes;
+all_codes_neu;
+alle_teiler;
+all_inj_codes;
+all_irred_polynomials;
+allkostka;
+all_lines;
+all_lyndon_words;
+all_orbits_set_rankf;
+all_orbits_set_trace;
+all_orbits_unset_rankf;
+all_orbits_unset_trace;
+all_orbits;
+all_plactic_word;
+all_points;
+all_points_phg;
+all_ppoly;
+all_weintrauben;
+alt_dimension;
+alt_odg_trafo;
+alt_sdg_trafo;
+anfang;
+an_odg;
+an_rz_perm;
+an_sdg;
+an_tafel;
+an_trafo_odg;
+an_trafo_sdg;
+append;
+append_apply;
+append_apply_part;
+append_apply_vector;
+append_behind_matrix_matrix;
+append_below_matrix_matrix;
+append_column_matrix;
+append_part_part;
+append_vector;
+apply_INJDT;
+augpart;
+bar_rectr;
+basis_mod_dg;
+bdg;
+b_d_sc;
+bestimme_D;
+bestimme_fixpunkt;
+bestimme_konjugiertenklasse;
+bestimme_zufallsmatrizen;
+b_gk_spa;
+bideterminant;
+bideterminant_tableaux;
+bideterminant_vector;
+bilde_htupel;
+bin_ggt;
+binom;
+binom_small;
+b_i_pa;
+bit;
+bit_longint;
+b_kl_pa;
+b_ksd_n;
+b_ks_o;
+b_ks_pa;
+b_ks_p;
+b_lh_m;
+b_lh_nm;
+b_lhs_m;
+b_l_nv;
+b_ls_v;
+b_l_v;
+b_matrix_tableaux;
+b_ou_b;
+b_o_v;
+b_pa_mon;
+b_pa_s;
+b_perm_vector_kranz;
+brauer_char;
+brouwerlowerbound;
+brouwerupperbound;
+bruch_anfang;
+bruch_ende;
+bruch_not_scalar;
+bru_comp;
+bruhat_comp_perm;
+bruhat_ideal;
+bruhat_ideal_strong;
+bruhat_ideal_weak;
+bruhat_interval_strong;
+bruhat_interval_weak;
+bruhat_rank_function;
+b_scalar_elmsym;
+b_scalar_homsym;
+b_scalar_monomial;
+b_scalar_powsym;
+b_scalar_schur;
+b_sk_mo;
+b_skn_mp;
+b_skn_po;
+b_skn_s;
+b_skn_sch;
+b_sn_e;
+b_sn_h;
+b_sn_l;
+b_sn_mon;
+b_sn_po;
+b_sn_ps;
+b_sn_s;
+b_s_po;
+build_gls_dist;
+build_lc;
+build_propab_vector;
+b_us_t;
+b_u_t;
+B_W;
+b_wpd_sc;
+calculate_fixed_point_number;
+callocobject;
+callocobject_anfang;
+callocobject_ende;
+callocobject_fast;
+callocobject_magma;
+cardinality_pgkq;
+cast_apply_barperm;
+cast_apply_bruch;
+cast_apply_elmsym;
+cast_apply_ff;
+cast_apply_homsym;
+cast_apply_integer;
+cast_apply;
+cast_apply_matrix;
+cast_apply_monom;
+cast_apply_monomial;
+cast_apply_monopoly;
+cast_apply_part;
+cast_apply_perm;
+cast_apply_polynom;
+cast_apply_powsym;
+cast_apply_schur;
+cast_apply_schubert;
+cast_apply_tableaux;
+cast_elmsym;
+cast_homsym;
+cast_monomial;
+cast_powsym;
+cast_schur;
+c_AUGPART_PARTITION;
+c_b_o;
+c_b_u;
+c_CHARAUGPART_CHARPARTITION;
+c_CHARPARTITION_CHARAUGPART;
+cc_muir_mms_partition_partition_;
+cc_plet_pes_integer_partition;
+cc_plet_phs_integer_partition;
+cc_plet_pss_integer_partition;
+cc_plet_pss_partition_partition;
+Cdeg;
+c_ff_di;
+c_ff_ip;
+Cgen;
+c_gr_k;
+c_gr_s;
+change_column_ij;
+change_row_ij;
+characteristic_polynom_faster;
+characteristic_polynom;
+characteristic_polynom_superfast;
+characteristik_symchar;
+characteristik_to_symchar;
+character_polynom;
+charakteristik_to_ypolynom;
+charge_tableaux;
+charge_word;
+char_matrix_scalar_product;
+chartafel;
+chartafel_bit;
+chartafel_nonbit;
+chartafel_partvector;
+chartafel_symfunc;
+charvalue_bit;
+charvalue;
+charvalue_tafel_part;
+check_braid;
+check_commute;
+check_equal_2a;
+check_equal_2;
+check_equal_3;
+check_equal_4;
+check_hecke_generators;
+check_hecke_quadratic;
+check_longint;
+check_result_0;
+check_result_1;
+check_result_2;
+check_result_3;
+check_result_5;
+check_selforthogonal_generatormatrix;
+check_time;
+check_zeilenposition;
+check_zero_matrix;
+c_i_i;
+c_ijk_sn;
+c_ijk_sn_tafel;
+c_ij_sn;
+c_i_n_an;
+c_i_n;
+c_kr_g;
+c_kr_v;
+class_bar;
+class_label_glnq_co;
+class_label_glnq;
+class_label;
+class_mult;
+class_mult_part_part;
+class_mult_schurmonom;
+class_mult_schur;
+class;
+class_rep_bar;
+class_rep_kranz;
+class_rep;
+c_l_n;
+clone_size_hashtable;
+c_l_s;
+c_m_h;
+c_m_hash;
+c_m_l;
+c_mo_k;
+c_mo_s;
+c_m_s;
+c_n_d;
+c_n_s;
+co_070295;
+code_mod_into_ord2;
+code_mod_into_ord;
+coeff_of_in;
+c_o_k;
+co_k_dimmod;
+columns_standardise_tableau;
+column_standardise_tableau;
+columnwordoftableaux;
+comp;
+comp_bigr_bigr;
+comp_bigr_perm;
+comp_bruch;
+comp_bruch_scalar;
+comp_bv;
+comp_colex_part;
+comp_colex_schurmonom;
+comp_colex_vector;
+comp_cyclo;
+comp_ff;
+comp_galois;
+comp_integer;
+comp_integer_integer;
+comp_integermatrix;
+comp_integervector;
+comp_kranztafel;
+complete_complete_plet;
+complete_schur_plet;
+complex_conjugate;
+comp_lex_perm;
+comp_list;
+comp_list_co;
+comp_longint;
+comp_longint_integer;
+comp_matrix;
+comp_monom;
+comp_monomelmsym;
+comp_monomhomsym;
+comp_monommonomial;
+comp_monompowsym;
+comp_monomschur;
+comp_monomvector_monomvector;
+comp_monopoly;
+comp_number;
+comp_numeric_vector;
+comp_partition;
+comp_partition_partition;
+comp_permutation;
+comp_permutation_pol;
+comp_polynom;
+comp_polynom_scalar;
+comp_reihe;
+comp_skewpartition;
+comp_skewpartition_skewpartition;
+comp_sqrad;
+comp_symchar;
+comp_tableaux;
+compute_complete_with_alphabet;
+compute_elmsym_with_alphabet;
+compute_gl_charvalue;
+compute_gl_c_ijk;
+compute_gl_cl_classorder;
+compute_gl_il_dimension;
+compute_monomial_with_alphabet;
+compute_power_with_alphabet;
+compute_schur;
+compute_schur_with_alphabet_det;
+compute_schur_with_alphabet;
+compute_skewschur_with_alphabet_det;
+compute_zonal_with_alphabet;
+comp_vector;
+comp_word;
+conj_cyclo;
+conj_sqrad;
+conjugate;
+conjugate_elmsym;
+conjugate_homsym;
+conjugate_monomial;
+conjugate_partition;
+conjugate_powsym;
+conjugate_schur;
+conjugate_tableaux;
+conjugation;
+consp_polynom;
+contain_comp_part;
+content;
+content_polynom;
+content_tableaux;
+content_word;
+convert_cyclo_scalar;
+convert_radical_cyclo;
+convert_sqrad_cyclo;
+convert_sqrad_scalar;
+co_polya3_sub;
+co_polya_sub;
+copy;
+copy_bintree;
+copy_bitvector;
+copy_bruch;
+copy_composition;
+copy_elmsym;
+copy_ff;
+copy_galois;
+copy_graph;
+copy_hashtable;
+copy_homsym;
+copy_integer;
+copy_integermatrix;
+copy_integervector;
+copy_kranztypus;
+copy_kranz;
+copy_laurent;
+copy_list;
+copy_longint;
+copy_matrix;
+copy_monom;
+copy_monomial;
+copy_number;
+copy_partition;
+copy_permutation;
+copy_polynom;
+copy_powsym;
+copy_queue;
+copy_reihe;
+copy_schur;
+copy_skewpartition;
+copy_subset;
+copy_symchar;
+copy_tableaux;
+copy_vector;
+copy_word;
+c_o_s;
+Cosinus_eingabe;
+co_table_of_R_nkq;
+co_zykelind_pglkq;
+c_pa_hash;
+c_pa_k;
+c_PARTITION_AUGPART;
+c_pa_s;
+c_p_k;
+c_p_s;
+c_sc_d;
+c_sch_n;
+c_sc_p;
+c_sc_w;
+c_s_n;
+c_spa_g;
+c_spa_k;
+c_t_s;
+c_t_u;
+c_v_i;
+c_v_l;
+c_v_s;
+cycliccodes_poly_matrix;
+cyclic_tafel;
+cyclo_an_tafel;
+cyclo_odg;
+debruijn_all_functions;
+debruijn_inj_functions;
+debugprint;
+debugprint_ff;
+debugprint_longint;
+debugprint_object;
+debugprint_reihe;
+dec;
+dec_integer;
+dec_integervector;
+dec_longint;
+decompose_primepower;
+dec_partition;
+dec_permutation;
+decp_mat;
+decreasingp_vector;
+dec_vector;
+degree_monopoly;
+degree_polynom;
+delete_column_matrix;
+delete_entry_vector;
+delete_row_matrix;
+derivative;
+det270588;
+det;
+det_imm_matrix;
+det_mat_imm;
+det_matrix;
+det_mat_tri;
+dg_mp;
+diagramm_permutation;
+dimension_augpart;
+dimension_bit;
+dimension_mod;
+dimension;
+dimension_partition;
+dimension_schubert;
+dimension_schur;
+dimension_skewpartition;
+dimension_symmetrization;
+dimino;
+display;
+display_schubert;
+distanzdiagramm;
+SYM_div;
+div_apply;
+div_apply_integer;
+div_default;
+divdiff;
+divdiff_bar;
+divdiff_perm_schubert;
+divdiff_schubert;
+divideddifference_bar;
+divideddifference;
+divideddiff_lc;
+divideddiff_permutation;
+divideddiff_rz_bar;
+divideddiff_rz;
+dixonwilf_bincodes;
+dixonwilf_primcodes;
+dixon_wilf_transversal;
+dom_comp_part;
+double_apply;
+double_apply_default;
+double_apply_longint;
+double_hashtable;
+ds_j_wt;
+durfee_size_part;
+dynamicp;
+dz_i_wt;
+E_eingabe;
+eins;
+eins_default;
+eins_ff;
+eins_ff_given_q;
+eins_galois;
+eins_gr_given_c_d;
+einsp;
+einsp_bitvector;
+einsp_bruch;
+einsp_cyclotomic;
+einsp_ff;
+einsp_galois;
+einsp_integer;
+einsp_integervector;
+einsp_kranz;
+einsp_longint;
+einsp_matrix;
+einsp_monopoly;
+einsp_permutation;
+einsp_polynom;
+einsp_reihe;
+einsp_schubert;
+einsp_sqrad;
+einsp_symchar;
+einsp_symfunc;
+einsp_vector;
+elementarp_permutation;
+elementary_schur_plet;
+embedding_mod_into_ord;
+empty_listp;
+empty_object;
+emptyp;
+ende;
+en_forme;
+enter_list_to_matrix;
+eq;
+eq_cyclotomic;
+eq_fieldobject_int;
+eq_integer;
+eq_integervector_integervector;
+eq_longint_longint;
+eq_matrix;
+eq_monomsymfunc;
+eq_monomsymfunchash;
+eq_partition;
+eq_partition_partition;
+eq_permutation;
+eq_sqrad;
+equal_2_error;
+equal_parts;
+eq_vector;
+eqv;
+error_during_computation_code;
+error_during_computation;
+error;
+ersetze_zeile;
+euler_phi;
+eval_2schubert;
+eval_char_polynom;
+eval_monopoly;
+eval_polynom_dir_prod;
+eval_polynom;
+even;
+even_integer;
+even_longint;
+even_partition;
+even_permutation;
+exchange_alphabets;
+exor_bitvector_apply;
+Exp_eingabe;
+extended_ggt;
+factorize;
+factorize_integer;
+fakul_longintresult;
+fakul;
+fastrectr;
+fatal_error;
+ferrers;
+ferrers_partition;
+ferrers_skewpartition;
+ff_anfang;
+ff_ende;
+fill_left_down_matrix;
+filter_apply_hashtable;
+filter_apply_list;
+filter_list;
+find_1result_hashtable;
+find_2result_hashtable;
+find;
+find_bintree;
+find_code_given_nkqd;
+find_code_given_nkqd_gen_tim_bou;
+find_hashtable;
+find_knuth_tab_entry;
+findmax_elmsym;
+findmax_homsym;
+findmax_monomial;
+findmax_powsym;
+findmax_schur;
+findmax_vector;
+findmin_elmsym;
+findmin_homsym;
+findmin_monomial;
+findmin_powsym;
+findmin_schur;
+find_monomial;
+find_non_root_standard_pos;
+find_non_rowstandard_pos;
+find_schur;
+find_tab_entry;
+find_teh_integer;
+find_tem_integer;
+find_tep_integer;
+find_the_integer;
+find_thm_integer;
+find_thp_integer;
+find_tme_integer;
+find_tmh_integer;
+find_tpe_integer;
+find_tph_integer;
+find_user_bintree;
+find_vector;
+first_bar;
+first_composition;
+first_ff;
+first_ff_given_q;
+first_gfp_vector;
+first_gr_given_c_d;
+first;
+first_kranztypus;
+first_lehmercode;
+first_lex_tableaux;
+first_part_EXPONENT;
+first_partition;
+first_part_VECTOR;
+first_perm_n_invers;
+first_permutation;
+first_pgp_vector;
+first_prime_factor;
+first_subset;
+first_tableaux;
+first_tab_perm;
+fprint_bintree;
+fprint_bitvector;
+fprint_bm;
+fprint_bruch;
+fprint_ff;
+fprint_graph;
+fprint_hashtable;
+fprint_integer;
+fprint_list;
+fprintln;
+fprint_longint;
+fprint_matrix;
+fprint_monom;
+fprint_number;
+fprint;
+fprint_partition;
+fprint_permutation;
+fprint_queue;
+fprint_reihe;
+fprint_skewpartition;
+fprint_symchar;
+fprint_tableaux;
+fprint_vector;
+freeall;
+freeall_magma;
+free_cyclotomic_parameters;
+freemonom;
+freepartition;
+free_root_parameters;
+freeself;
+freeself_bintree;
+freeself_bitvector;
+freeself_bruch;
+freeself_ff;
+freeself_galois;
+freeself_graph;
+freeself_hashtable;
+freeself_integer;
+freeself_integermatrix;
+freeself_integervector;
+freeself_kranz;
+freeself_kranztypus;
+freeself_laurent;
+freeself_list;
+freeself_longint;
+freeself_matrix;
+freeself_monom;
+freeself_number;
+freeself_partition;
+freeself_permutation;
+freeself_reihe;
+freeself_skewpartition;
+freeself_symchar;
+freeself_tableaux;
+freeself_vector;
+free_useful_monopolies;
+freevectorstruct;
+frip_latex_matrix_to_table;
+frip_latex_zykelind;
+frobenius_elmsym;
+frobenius_homsym;
+frobenius_monomial;
+frobenius_powsym;
+frobenius_schur;
+from_loesung_to_generatormatrix;
+from_loesung_to_maximaldistanz;
+from_loesung_to_minimaldistanz;
+from_loesung_to_weightenumerator;
+fusedmemory;
+galois_anfang;
+galois_ende;
+ganzdiv;
+ganzdiv_apply;
+ganzdiv_apply_integer;
+ganzdiv_apply_longint;
+ganzdiv_apply_longint_integer;
+ganzdiv_apply_longint_longint;
+ganzdiv_integer;
+ganzdiv_integer_integer;
+ganzdiv_integer_longint;
+ganzdiv_longint;
+ganzdiv_longint_integer;
+ganzdiv_longint_longint;
+ganzsquareroot;
+ganzsquareroot_integer;
+ganzsquareroot_longint;
+garnir;
+gauss_form;
+gauss_numbers;
+gauss_polynom;
+GaussRecInternal;
+gauss_schubert_polynom;
+gauss_triangular_apply_co;
+gauss_triangular_apply_nodiv;
+gauss_triangular_apply;
+gcd_ex;
+gcd_int_po;
+gcd_mp;
+gcd_mp_lent;
+ge;
+generate_root_tableaux;
+generate_standard_tableaux;
+generator_glkq_singer;
+generator_glkq_sn;
+generators_glnq;
+generators_slnq;
+gengroup;
+gen_mat;
+genpoly_QRcode;
+gen_smat;
+get_ax_dist;
+get_bm_ij;
+get_bv_i;
+get_ff_irred;
+get_galois_irred;
+get_incidence;
+get_incidence_caps;
+get_incidence_phg;
+get_incidence_set_maxorbits;
+get_index;
+get_level_vector_of_verband;
+get_orb_rep;
+get_perm;
+get_position;
+get_tex_polynom_parameter;
+get_time;
+gewicht_v;
+gewicht_wt;
+Ggen;
+ggt;
+ggt_field_polynom;
+ggt_i;
+ggt_integer;
+ggt_integer_integer;
+ggt_integer_integer_slow;
+ggt_integer_longint;
+ggt_integer_slow;
+ggt_longint;
+ggt_longint_integer;
+ggt_longint_longint;
+ggt_longint_longint_sub;
+ggt_polynom;
+giambelli_matrix;
+gl_character;
+gl_dimension;
+glm_B_W;
+glm_get_BV;
+glm_homtest;
+glmndg;
+glm_sab;
+glpdg;
+gl_tableaux;
+gr;
+gram_schmidt;
+grf_An;
+grf_arb;
+grf_Cn;
+grf_Dn;
+grf_Sn;
+griesmerbound;
+group_gen;
+grouporder_kranz;
+growingorder_schur;
+gt;
+gupta_nm;
+gupta_nm_speicher;
+gupta_tafel;
+half_apply;
+half_apply_longint;
+hall_littlewood;
+hall_littlewood_alt;
+hall_littlewood_dij;
+hall_littlewood_tafel;
+hamming_distance_vector;
+hash;
+hash_ende;
+hash_ff;
+hash_integervector;
+hash_list;
+hash_matrix;
+hash_monompartition;
+hash_partition;
+hash_skewpartition;
+hashv;
+has_one_variable;
+hecke_action_lc_on_lc;
+hecke_action_perm_on_hecke;
+hecke_action_perm_on_lc;
+hecke_add;
+hecke_dg;
+hecke_generator_reps;
+hecke_mult;
+hecke_root_generator_reps;
+hecke_scale;
+hfix_in_ww;
+hoch;
+hoch_bruch;
+hoch_bruch_integer;
+hoch_bruch_longint;
+hoch_default;
+hoch_integer;
+hoch_integer_integer;
+hoch_integer_longint;
+hoch_longint;
+hoch_longint_integer;
+hoch_longint_longint;
+hoch_pre200902;
+_homtest;
+hook_diagramm;
+hook_length_augpart;
+hook_length;
+hookp;
+hook_partition;
+hook_part;
+horizontal_sum;
+hplus;
+idempotent;
+immanente_matrix;
+inc;
+inc_bitvector;
+inc_integer;
+inc_longint;
+inc_matrix;
+inc_matrix_column_co;
+inc_matrix_row_co;
+inc_partition;
+inc_permutation;
+inc_reihe;
+inc_tableaux;
+inc_vector;
+inc_vector_co;
+index_galois;
+indexofpart;
+index_vector;
+index_vector_binary;
+inf_bitvector_apply;
+inf_bitvector;
+init_bintree;
+init_cyclo;
+init_elmsym;
+init_galois_global;
+init_hall_littlewood;
+init_hashtable;
+init_homsym;
+init;
+init_kostka;
+init_kranz;
+init_longint;
+init_monomial;
+init_polynom;
+init_powsym;
+init_queue;
+init_reihe;
+init_schur;
+init_size_hashtable;
+init_sqrad;
+innermaxmofn;
+inner_tensor_sc;
+input_glmn;
+input_lc_permutations;
+input_tableau;
+insert;
+insert_bintree;
+insert_bt_bt;
+insert_elmsym_hashtable;
+insert_entry_vector;
+insert_hashtable;
+insert_hashtable_hashtable;
+insert_homsym_hashtable;
+insert_list_list_2;
+insert_list_list;
+insert_list;
+insert_monomial_hashtable;
+insert_powsym_hashtable;
+insert_scalar_hashtable;
+insert_schur_hashtable;
+integer_factor;
+integer_factors_to_integer;
+integer_primep;
+intlog;
+intlog_int;
+intlog_longint;
+invers;
+invers_apply;
+invers_apply_bruch;
+invers_apply_integer;
+invers_apply_longint;
+invers_apply_ff;
+invers_bar;
+invers_bitvector;
+invers_bruch;
+invers_cyclo;
+inverse_jeudetaquin_tableaux;
+inverse_nilplactic_jeudetaquin_tableaux;
+invers_ff;
+invers_galois;
+invers_integer;
+inversion_matrix_perm;
+invers_kostka_tafel;
+invers_kranz;
+invers_laurent;
+invers_longint;
+invers_matrix;
+invers_monopoly;
+inversordcen;
+invers_permutation;
+invers_polynom;
+invers_POLYNOM;
+invers_sqrad;
+is_graphical;
+is_scalar_polynom;
+is_scalar_reihe;
+is_selfconjugate;
+jacobi;
+jacobitrudimatrix;
+jeudetaquin_tableaux;
+johnson_graph_adjacency_matrix;
+kgv;
+kk_280604;
+Kn_adjacency_matrix;
+knuth_row_delete_step;
+knuth_row_insert_step;
+knuth_twoword;
+konj_gral_perm;
+konj_perm_perm;
+konjugation2;
+konjugation;
+konjugierende;
+konjugiertenklassen_glkp;
+kostka_character;
+kostka_number;
+kostka_number_partition;
+kostka_number_skewpartition;
+kostka_tab;
+kostka_tafel;
+kranztafel;
+kranztypus_charakteristik;
+kranztypus_kranztypus_monom;
+kranztypus_to_matrix;
+kronecker;
+kronecker_product;
+krz;
+kuerzen;
+kuerzen_integer_integer;
+kuerzen_integer_longint;
+kuerzen_integral;
+kuerzen_longint_integer;
+kuerzen_longint_longint;
+kung_formel;
+lagrange_polynom;
+last_lehmercode;
+lastof;
+lastof_integervector;
+lastof_partition;
+lastof_skewpartition;
+lastof_vector;
+lastp;
+last_part_EXPONENT;
+last_partition;
+last_part_VECTOR;
+last_permutation;
+lastp_list;
+latex_glm_dar;
+latex_kranztafel;
+latex_line;
+latex_verband;
+latticepword;
+l_complete_complete_plet;
+l_complete_schur_plet;
+ldcf_monopoly;
+ldcf_mp;
+ldrei_neu;
+ldrei_neu_get_zeitlimit;
+ldrei_neu_set_anzahl;
+ldrei_neu_set_keepgls;
+ldrei_neu_set_mckay;
+ldrei_neu_set_zeitlimit;
+le;
+leftkey_wt;
+lehmercode2_permutation;
+lehmercode;
+lehmercode_bar;
+lehmercode_permutation;
+lehmercode_tableaux;
+lehmercode_vector_bar;
+lehmercode_vector;
+l_elementary_schur_plet;
+length;
+length_bar;
+length_bintree;
+length_comp_part;
+length_list;
+length_partition;
+length_permutation;
+length_reihe;
+length_skewpartition;
+length_vector;
+line_of_two_points;
+list_anfang;
+list_ende;
+listp;
+local_anfang;
+local_ende;
+longint_ende;
+l_outerproduct_schur_lrs;
+l_powerproduct_schur_plet;
+l_power_schur_plet;
+l_schur_powerproduct_schur_plet_mult;
+l_schur_power_schur_plet_mult;
+l_schur_schur_plet;
+l_schur_schur_pletbis;
+lt;
+lyndon_orb;
+m_2schubert_wt;
+mahh_hashtable_hashtable_;
+mahh_integer_homsym_;
+mahh_partition_hashtable_;
+make_all_st_tabs;
+make_alt_classes;
+makealtclassreps;
+make_alt_partitions;
+make_coprimes;
+make_cyclotomic_monopoly;
+make_ij_part;
+make_index_coeff_power_cyclo;
+make_index_power_cyclo;
+make_monopoly_sqrad;
+make_neu_partij_schur;
+make_n_id;
+make_n_kelmtrans;
+make_n_transpositionmatrix;
+make_nzykel;
+make_partij_perm;
+make_partij_schur;
+make_real_cycletype;
+make_scalar_cyclo;
+make_scalar_sqrad;
+make_tab_signs;
+make_unitary0_monopoly;
+make_unitary_eins_monopoly;
+makevectorof_class_bar;
+makevectorof_class_rep_bar;
+makevectorof_kranztypus;
+makevectorofpart_EXPONENT;
+makevectorofpart;
+makevectorofperm;
+makevectorofrect_lehmercode;
+makevectorofrect_permutation;
+makevectorofshuffle;
+makevectorofspecht_poly;
+makevectorofsubsets;
+makevectorofsymspecht_poly;
+makevectorofSYT;
+makevectoroftableaux;
+makevectoroftranspositions;
+makevectorofvectors;
+makevectorofwords;
+maple_polynom;
+mapp_hashtable_hashtable_;
+matrix_knuth;
+matrix_monom_ypolynom;
+matrixp;
+matrix_to_kranztypus;
+matrix_twoword;
+max;
+max_bar;
+max_degree_reihe;
+maxdegree_schubert;
+max_divideddiff;
+max_integervector;
+max_matrix;
+maxorder_young;
+maxpart_comp_part;
+maxrindexword;
+max_tableaux;
+max_vector;
+m_bar_schubert;
+m_cosinus_reihe;
+m_d_sc;
+mee_elmsym__;
+mee_hashtable__;
+mee_hashtable_hashtable_;
+mee_integer__;
+mee_integer_hashtable_;
+mee_integer_partition_;
+mee_partition__;
+mee_partition_partition_;
+m_eins_reihe;
+memcheck;
+mem_elmsym__;
+mem_ende;
+mem_integer__;
+mem_integer_hashtable_;
+mem_integer_partition_;
+mem_partition__;
+mem_size;
+mem_size_hashtable;
+mem_size_longint;
+mem_size_matrix;
+mem_size_vector;
+mem_small;
+mes_elmsym__;
+mes_ende;
+mes_hashtable__;
+mes_integer__;
+mes_integer_hashtable_;
+mes_integer_partition_;
+mes_partition__;
+m_ff_vector;
+m_forall_monomials_in_a;
+m_forall_monomials_in_ab;
+m_forall_monomials_in_b;
+m_function_reihe;
+m_gk_spa;
+m_gl_alt;
+m_gl_chartafel;
+m_gl_cl;
+m_gl_co;
+m_gl_cr;
+m_gl_cyclic;
+m_gl_first;
+m_gl_ge_cl;
+m_gl_glnq;
+m_gl_go;
+m_gl_hyp;
+m_gl_il;
+m_gl_nc;
+m_gl_next;
+m_gl_sym;
+m_gl_symkranz;
+mhh_hashtable__;
+mhh_hashtable_hashtable_;
+mhh_homsym__;
+mhh_partition__;
+mhh_partition_hashtable_;
+mhh_partition_partition_;
+mhm_hashtable__;
+mhm_homsym__;
+mhm_integer__;
+mhm_integer_hashtable_hashtable;
+mhm_integer_partition_hashtable;
+mhm_null__;
+mhm_partition__;
+mhp_co;
+mhp_hashtable__;
+mhp_homsym__;
+mhp_integer__;
+mhp_integer_hashtable_;
+mhp_integer_partition_;
+mhp_partition__;
+mhs_hashtable__;
+mhs_homsym__;
+mhs_integer__;
+mhs_integer_partition_;
+mhs_partition__;
+m_i_elmsym;
+m_i_i;
+m_iindex_iexponent_monom;
+m_iindex_monom;
+m_iindex_monopoly;
+m_il_bv;
+m_ilih_bm;
+m_ilih_m;
+m_ilih_nm;
+m_il_integervector;
+m_il_nbv;
+m_il_nv;
+m_i_longint;
+m_il_pa;
+m_il_p;
+m_il_v;
+min;
+m_index_monom;
+mindist_neu;
+mindist;
+mindist_submatrix;
+minimaldistanz;
+minimal_extension;
+minimum_distance;
+min_integervector;
+min_matrix;
+min_tableaux;
+m_INTEGER_elmtrans;
+m_INT_elmtrans;
+m_int_pa;
+min_vector;
+m_ioiu_b;
+m_i_pa;
+m_i_powsym;
+m_i_schubert;
+m_i_schur;
+m_i_staircase;
+m_kl_pa;
+m_ksd_n;
+m_ks_pa;
+m_ks_p;
+m_lehmer_schubert_monom_summe;
+m_lehmer_schubert_qpolynom;
+m_lh_m;
+m_lh_nm;
+m_l_nv;
+m_l_p;
+m_l_v;
+m_matrix_tableaux;
+m_matrix_umriss;
+m_merge_partition_partition;
+mmm___;
+mmm_ende;
+mmm_hashtable__;
+mmm_hashtable_hashtable_;
+mmm_integer__;
+mmm_integer_hashtable_;
+mmm_integer_partition_;
+mmm_monomial__;
+mmm_null_partition_;
+mmm_partition__;
+mmm_partition_partition_;
+mms___;
+mms_hashtable__;
+mms_hashtable_partition_;
+mms_integer__;
+mms_integer_partition_;
+mms_monomial__;
+mms_null__;
+mms_partition__;
+mms_partition_partition_;
+m_nc_kranz;
+mod;
+mod_apply;
+mod_apply_integer;
+mod_apply_integer_longint;
+mod_apply_longint;
+moddg;
+mod_dg_sbd;
+mod_integer_integer;
+mod_longint_integer;
+mod_longint_integer_via_ganzsquores;
+mod_matrix;
+mod_monom;
+mod_part;
+mod_polynom;
+mod_vector;
+moebius;
+moebius_tafel;
+monom_anfang;
+monom_ende;
+monomial_recursion;
+monomial_recursion2;
+monom_release;
+m_ou_b;
+move_1result_hashtable;
+move_2result_hashtable;
+m_o_v;
+m_pa_e;
+m_pa_h;
+m_pa_mon;
+m_pa_ps;
+m_part_centralsc;
+m_part_kostkaperm;
+m_part_part_perm;
+m_part_perm;
+m_part_qelm;
+m_part_Qschur;
+m_part_sc;
+m_part_sc_tafel;
+m_part_tableaux;
+m_part_youngsc;
+m_pa_s;
+m_perm_2schubert_monom_summe;
+m_perm_2schubert_operating_monom_summe;
+m_perm_2wt_summe;
+m_perm_paareperm;
+m_perm_reihe;
+m_perm_rz_number;
+m_perm_rz_set;
+m_perm_sch;
+m_perm_schubert_dimension;
+m_perm_schubert_monom_summe;
+m_perm_schubert_qpolynom;
+m_perm_schur;
+m_perm_sdwt_summe;
+m_perm_skwt_summe;
+m_perm_snwt_summe;
+m_perm_tableaux_summe;
+m_perm_wt_summe;
+mp_is_cst;
+mpp___;
+mpp_ende;
+mpp_hashtable__;
+mpp_hashtable_hashtable_;
+mpp_integer__;
+mpp_integer_hashtable_;
+mpp_integer_partition_;
+mpp_partition__;
+mpp_partition_partition_;
+mpp_powsym__;
+mps___;
+mps_ende;
+mps_hashtable__;
+mps_hashtable__pre100102;
+mps_integer__;
+mps_integer_partition_;
+mps_null__;
+mps_partition__;
+mps_powsym__;
+m_scalar_bruch;
+m_scalar_elmsym;
+m_scalar_homsym;
+m_scalar_monomial;
+m_scalar_polynom;
+m_scalar_powsym;
+m_scalar_reihe;
+m_scalar_schubert;
+m_scalar_schur;
+m_sinus_reihe;
+m_skewpart_skewperm;
+m_sk_gr;
+m_sk_mo;
+m_skn_mp;
+m_skn_po;
+m_skn_sch;
+m_skn_s;
+m_sn_l;
+m_s_po;
+mss___;
+mss_ende;
+mss_hashtable__;
+mss_hashtable_hashtable_;
+mss_hashtable_hashtable_maxpart_maxlength;
+mss_hashtable__maxpart_maxlength;
+mss___maxpart_maxlength;
+mss_partition__;
+mss_partition__maxpart_maxlength;
+mss_partition_partition_;
+mss_partition_partition_maxpart_maxlength;
+mss_schur__;
+mss_schur__maxpart_maxlength;
+m_tableaux_polynom;
+m_tableaux_tableauxpair;
+m_T_pol_summe;
+m_T_tableaux_summe;
+m_T_wt_summe;
+mult;
+multadd_apply;
+multadd_apply_default;
+multadd_apply_ff;
+mult_apply;
+mult_apply_bruch;
+mult_apply_bruch_bruch;
+mult_apply_bruch_hashtable;
+mult_apply_bruch_integer;
+mult_apply_bruch_longint;
+mult_apply_bruch_monom;
+mult_apply_bruch_polynom;
+mult_apply_bruch_scalar;
+mult_apply_cyclo;
+mult_apply_default;
+mult_apply_elmsym;
+mult_apply_elmsym_elmsym;
+mult_apply_ff;
+mult_apply_gral;
+mult_apply_homsym;
+mult_apply_homsym_homsym;
+mult_apply_homsym_schur;
+mult_apply_integer;
+mult_apply_integer_bruch;
+mult_apply_integer_hashtable;
+mult_apply_integer_integer;
+mult_apply_integer_integervector;
+mult_apply_integer_longint;
+mult_apply_integer_matrix;
+mult_apply_integer_monom;
+mult_apply_integer_polynom;
+mult_apply_longint;
+mult_apply_longint_bruch;
+mult_apply_longint_integer;
+mult_apply_longint_longint;
+mult_apply_longint_matrix;
+mult_apply_longint_polynom;
+mult_apply_matrix;
+mult_apply_matrix_matrix;
+mult_apply_monom;
+mult_apply_monomial;
+mult_apply_monomial_monomial;
+mult_apply_monopoly;
+mult_apply_permutation;
+mult_apply_polynom;
+mult_apply_polynom_polynom;
+mult_apply_polynom_scalar;
+mult_apply_powsym;
+mult_apply_powsym_powsym;
+mult_apply_reihe;
+mult_apply_scalar_bruch;
+mult_apply_scalar_cyclo;
+mult_apply_scalar_hashtable;
+mult_apply_scalar_matrix;
+mult_apply_scalar_monom;
+mult_apply_scalar_monopoly;
+mult_apply_scalar_polynom;
+mult_apply_scalar_sqrad;
+mult_apply_scalar_symchar;
+mult_apply_scalar_vector;
+mult_apply_schur;
+mult_apply_schur_schur;
+mult_apply_sqrad;
+mult_apply_symchar;
+mult_apply_vector;
+mult_apply_vector_vector;
+mult_bar_bar;
+mult_bruch;
+mult_bruch_bruch;
+mult_bruch_hashtable;
+mult_bruch_integer;
+mult_bruch_longint;
+mult_cyclo;
+mult_cyclo_cyclo;
+mult_disjunkt_polynom_polynom;
+mult_elmsym;
+mult_elmsym_elmsym;
+mult_elmsym_homsym;
+mult_elmsym_monomial;
+mult_elmsym_powsym;
+mult_elmsym_scalar;
+mult_elmsym_schur;
+mult_ff;
+mult_ff_ff;
+mult_ff_integer;
+mult_galois_galois;
+mult_galois;
+mult_gral;
+mult_gral_gral;
+mult_hashtable_hashtable;
+mult_hashtable_hashtable_faktor;
+mult_homsym;
+mult_homsym_elmsym;
+mult_homsym_homsym;
+mult_homsym_monomial;
+mult_homsym_powsym;
+mult_homsym_scalar;
+mult_homsym_schur;
+mult_imatrix_imatrix;
+multinom;
+multinom_small;
+mult_integer;
+mult_integer_bruch;
+mult_integer_hashtable;
+mult_integer_integer;
+mult_integer_longint;
+mult_integer_monom;
+multiplicity_part;
+mult_kranz_kranz;
+mult_laurent;
+mult_longint;
+mult_longint_integer;
+mult_longint_integer_via_ganzsmul;
+mult_longint_longint;
+mult_matrix;
+mult_matrix_matrix;
+mult_matrix_vector;
+mult_monom;
+mult_monomial;
+mult_monomial_elmsym;
+mult_monomial_homsym;
+mult_monomial_monomial;
+mult_monomial_powsym;
+mult_monomial_scalar;
+mult_monomial_schur;
+mult_monopoly;
+mult_monopoly_monopoly;
+mult_monopoly_polynom;
+mult_nc_kranz;
+mult_perm_fix;
+mult_permutation;
+mult_polynom;
+mult_polynom_polynom;
+mult_power_schur;
+mult_powsym;
+mult_powsym_elmsym;
+mult_powsym_homsym;
+mult_powsym_monomial;
+mult_powsym_powsym;
+mult_powsym_scalar;
+mult_powsym_schur;
+mult_reihe;
+mult_scalar_cyclo;
+mult_scalar_gral;
+mult_scalar_matrix;
+mult_scalar_monom;
+mult_scalar_monopoly;
+mult_scalar_polynom;
+mult_scalar_schubert;
+mult_scalar_sqrad;
+mult_scalar_symchar;
+mult_scalar_vector;
+mult_schubert_monom;
+mult_schubert_polynom;
+mult_schubert_schubert;
+mult_schubert_variable;
+mult_schur;
+mult_schur_elmsym;
+mult_schur_homsym;
+mult_schur_monomial;
+mult_schur_powsym;
+mult_schur_scalar;
+mult_schur_schur;
+mult_schur_schur_maxlength;
+mult_schur_schur_maxpart_maxlength;
+mult_sqrad;
+mult_sqrad_sqrad;
+mult_symchar_symchar;
+mult_vector_matrix;
+mult_vector_vector;
+m_umriss_tableaux;
+m_us_t;
+m_u_t;
+m_vcl_kranz;
+m_vco_kranz;
+m_vec_grad_nc_hyp;
+m_vec_poly;
+m_vector_diagonalmatrix;
+m_vector_ff;
+m_vector_graph;
+m_vec_vec_poly;
+m_v_pa;
+m_v_po_mz;
+m_v_s;
+m_word_wt;
+m_wpd_sc;
+m_wt_tableaux;
+m_wttab_tableaux;
+m_wt_wtkontertab;
+m_wt_wttab;
+mx_outerproduct_schur_lrs;
+mx_powerproduct_schur_plet;
+mx_power_schur_plet;
+mx_schur_powerproduct_schur_plet_mult;
+mx_schur_power_schur_plet_mult;
+mx_schur_schur_plet;
+mx_schur_schur_pletbis;
+mxx_null__;
+my_conjugation;
+my_quasinormalform_neu;
+mz_extrahieren;
+mz_vereinfachen;
+nachfolger_young;
+nb_ende;
+ndg;
+negeinsp;
+negeinsp_bruch;
+negeinsp_integer;
+negeinsp_longint;
+negeinsp_polynom;
+negp;
+negp_bruch;
+negp_integer;
+negp_longint;
+negp_polynom;
+neg_sum;
+neq;
+neqparts_partition;
+new_divdiff_bar;
+new_divideddifference_bar;
+new_divideddiff_rz_bar;
+new_orbit;
+newtrans_eins;
+newtrans_lehmer;
+newtrans_limitfunction;
+newtrans_limit_limitfunction;
+newtrans_maxpart_maxlength;
+newtrans_maxpart;
+newtrans;
+next_apply_bar;
+next_apply_composition;
+next_apply_ff;
+next_apply_gr;
+next_apply;
+next_apply_partition;
+next_apply_permutation;
+next_apply_subset;
+next_bar;
+next_composition;
+next_ff;
+next_gfp_vector;
+next_gr;
+next_kranztypus;
+next_lehmercode;
+next_lex_tableaux;
+next_lex_vector;
+next_part_EXPONENT_apply;
+next_part_EXPONENT;
+next_partition_apply;
+next_partition;
+next_part_VECTOR_apply;
+next_part_VECTOR;
+next_perm_invers;
+next_permutation;
+next_permutation_lex;
+next_pgp_vector;
+next_shuffle_part;
+next_shufflepermutation;
+next_shufflevector;
+next_subset;
+next;
+n_fold_kronecker_product;
+no_memory;
+no_orbits_arb;
+normal_laurent;
+not_yet_implemented;
+nth_root_ff_given_q;
+ntopaar_symchar;
+null;
+null_default;
+null_ff;
+null_ff_given_q;
+null_galois;
+null_gr_given_c_d;
+null_object;
+nullp;
+nullp_bitvector;
+nullp_bruch;
+nullp_cyclo;
+nullp_elmsym;
+nullp_ff;
+nullp_galois;
+nullp_homsym;
+nullp_integer;
+nullp_integermatrix;
+nullp_integervector;
+nullp_longint;
+nullp_matrix;
+nullp_monomial;
+nullp_monopoly;
+nullp_polynom;
+nullp_powsym;
+nullp_reihe;
+nullp_schubert;
+nullp_schur;
+nullp_sqrad;
+nullp_symchar;
+nullp_vector;
+number_01_matrices;
+number_nat_matrices;
+number_of_bits;
+numberof_class_kranz;
+number_of_digits;
+numberof_inversionen;
+number_of_irred_poly_of_degree;
+numberofmonomials;
+numberofpart_i;
+numberofpart;
+numberofparts_exact_parts;
+numberofparts_ge;
+numberofparts_le_parts;
+numberofselfconjugatepart;
+numberof_shufflepermutation;
+number_of_subspaces;
+numberofvariables;
+nu_v_v_pgp;
+nxt_ym;
+objectread_bruch;
+objectread_bv;
+objectread_ff;
+objectread;
+objectread_gral;
+objectread_hashtable;
+objectread_integer;
+objectread_list;
+objectread_longint;
+objectread_matrix;
+objectread_monom;
+objectread_monopoly;
+objectread_number;
+objectread_partition;
+objectread_permutation;
+objectread_schur;
+objectread_skewpartition;
+objectread_symchar;
+objectread_tableaux;
+objectread_vector;
+objectwrite_bruch;
+objectwrite_bv;
+objectwrite_ff;
+objectwrite;
+objectwrite_gral;
+objectwrite_hashtable;
+objectwrite_integer;
+objectwrite_list;
+objectwrite_longint;
+objectwrite_matrix;
+objectwrite_monom;
+objectwrite_number;
+objectwrite_partition;
+objectwrite_permutation;
+objectwrite_schur;
+objectwrite_skewpartition;
+objectwrite_symchar;
+objectwrite_tableaux;
+objectwrite_vector;
+odd;
+oddify_longint;
+odd_longint;
+oddpartsp;
+odd_to_strict_part;
+odg;
+old_number_01_matrices;
+operate_gral_polynom;
+operate_perm_bideterminant;
+operate_perm_polynom;
+operate_perm_spaltenmatrix;
+operate_perm_tableaux;
+operate_perm_vector;
+operate_perm_zeilenmatrix;
+orbit;
+orbit_set_max_size;
+orbit_words;
+orblen;
+or_character;
+ordcen_bar;
+ordcen;
+ordcon_bar;
+ordcon;
+order_class_kranz;
+order_ff;
+order_permutation;
+or_dimension;
+ordnung_affkq;
+ordnung_glkq;
+or_tableaux;
+outerproduct_schubert;
+outerproduct_schur;
+outerproduct_schur_limit;
+outerproduct_schur_limitfunction;
+outerproduct_schur_limit_limitfunction;
+outerproduct_schur_lrs;
+p2_schursum;
+part_anfang;
+part_comp;
+part_ende;
+partitionp;
+part_part_skewschur;
+perm_anfang;
+Perm_eingabe;
+perm_ende;
+perm_matrix;
+perm_matrix_p;
+perm_tableaux;
+permutation_matrix;
+permutationp;
+permutationsdarstellung_matrix_pgl;
+perm_weintraube;
+pes___;
+pes_elmsym__;
+pes_ende;
+pes_hashtable__;
+pes_integer__;
+pes_integer_hashtable_;
+pes_integer_partition_;
+pes_null__;
+pes_null_partition_;
+pes_partition__;
+pfact;
+pfaffian_matrix;
+pgcd;
+phm___;
+phm_ende;
+phm_hashtable__;
+phm_hashtable_hashtable_;
+phm_homsym__;
+phm_integer__;
+phm_integer_hashtable_;
+phm_integer_integer_;
+phm_integer_partition_;
+phm_null__;
+phm_null_partition_;
+phm_partition__;
+p_hook_diagramm;
+p_hook_weight;
+phs___;
+phs_ende;
+phs_hashtable__;
+phs_homsym__;
+phs_integer__;
+phs_integer_hashtable_;
+phs_integer_partition_;
+phs_null__;
+phs_null_partition_;
+phs_partition__;
+pictex_skwt_liste;
+pictex_snwt_liste;
+pictex_wt;
+pictex_wt_liste;
+pictex_wt_liste_co;
+planep;
+plane_tableau;
+plaziere_verband;
+plet_elmsym_elmsym;
+plet_elmsym_homsym;
+plet_elmsym_monomial;
+plet_elmsym_powsym;
+plet_elmsym_schur;
+plet_homsym_elmsym;
+plet_homsym_homsym;
+plet_homsym_monomial;
+plet_homsym_monomial_via_ppm;
+plet_homsym_powsym;
+plet_homsym_schur;
+plethysm;
+plethysm_schur_monomial;
+plethysm_schur_schur;
+plet_monomial_elmsym;
+plet_monomial_homsym;
+plet_monomial_monomial;
+plet_monomial_powsym;
+plet_monomial_schur;
+plet_powsym_elmsym;
+plet_powsym_homsym;
+plet_powsym_monomial;
+plet_powsym_powsym;
+plet_powsym_schur;
+plet_powsym_schur_via_ppm;
+plet_schur_elmsym;
+plet_schur_elmsym_pre101201;
+plet_schur_homsym;
+plet_schur_monomial;
+plet_schur_monomial_new;
+plet_schur_powsym;
+plet_schur_schur;
+plet_schur_schur_pol;
+plet_schur_schur_via_phs;
+pl_j_wt;
+pn_character;
+pn_dimension;
+pn_tableaux;
+point;
+pol_sd_wt;
+pol_sn_wt;
+pol_wt;
+polya1_sub;
+polya2_sub;
+polya3_sub;
+polya_const_sub;
+polya_multi_const_sub;
+polya_multi_sub;
+polya_n_sub;
+polya_sub;
+pop;
+posp;
+posp_bruch;
+posp_integer;
+posp_longint;
+posp_polynom;
+posp_vector;
+pos_sum;
+powerproduct_schur_plet;
+power_schur_plet;
+power_schur_plet_old;
+ppe___;
+ppe_ende;
+ppe_hashtable__;
+ppe_hashtable_hashtable_;
+ppe_integer__;
+ppe_integer_hashtable_;
+ppe_integer_integer_;
+ppe_integer_partition_;
+ppe_null__;
+ppe_null_partition_;
+ppe_partition__;
+ppe_powsym__;
+pph___;
+pph_ende;
+pph_hashtable__;
+pph_hashtable_hashtable_;
+pph_integer__;
+pph_integer_hashtable_;
+pph_integer_integer_;
+pph_integer_partition_;
+pph_null__;
+pph_null_partition_;
+pph_partition__;
+pph_powsym__;
+ppm___;
+ppm_ende;
+ppm_hashtable__;
+ppm_hashtable_hashtable_;
+ppm_integer__;
+ppm_integer_hashtable_;
+ppm_integer_integer_;
+ppm_integer_partition_;
+ppm_null__;
+ppm_null_partition_;
+ppm_partition__;
+ppm_powsym__;
+ppp___;
+ppp_ende;
+ppp_hashtable__;
+ppp_hashtable_hashtable_;
+ppp_integer__;
+ppp_integer_hashtable_;
+ppp_integer_partition_;
+ppp_null__;
+ppp_null_partition_;
+ppp_partition__;
+ppp_powsym__;
+pps___;
+pps_ende;
+pps_hashtable__;
+pps_hashtable_hashtable_;
+pps_integer__;
+pps_null__;
+pps_null_partition_;
+pps_partition__;
+pps_powsym__;
+prepartdom;
+primep;
+primep_ff;
+prime_power_p;
+primitive_element_ff_given_q;
+primitive_element_ff;
+print_cyclo_data;
+print_cyclo_list;
+print_cyclo_table;
+printeingabe;
+println_bm;
+println;
+println_schub_lehmer;
+printobjectkind;
+print;
+print_schubert_difference;
+print_stat_hashtable;
+print_time;
+print_type;
+pr_j_wt;
+p_root_part;
+p_root_schur;
+prsym;
+p_schursum;
+psh___;
+psh_ende;
+psh_hashtable__;
+psh_integer__;
+psh_integer_homsym_;
+psh_null__;
+psh_partition__;
+psh_partition_homsym_;
+psh_partition_partition_;
+psh_schur__;
+psl_apply_i_integer;
+psl_apply_i_longint;
+psl_apply_longint;
+psm___;
+psm_ende;
+p_splitpart;
+p_splitpart2;
+psr_apply_i_integer;
+psr_apply_i_longint;
+pss___;
+pss_ende;
+pss_hashtable__;
+pss_hashtable_hashtable_;
+pss_integer__;
+pss_integer_hashtable_;
+pss_integer_integer_;
+pss_integer_partition_;
+pss_integer_schur_;
+pss_null__;
+pss_null_partition_;
+pss_partition__;
+pss_partition_hashtable_;
+pss_partition_partition_;
+pss_partition_schur_;
+pss_schur__;
+P_symmetrized_bideterminant;
+push;
+put_exactlength_limit_schur;
+put_length_limit_schur;
+q_core;
+q_core_sign;
+qdimension;
+qdimension_schubert;
+qsort_vector;
+quadraticp;
+quadratic_remainders;
+quasi_normal_form;
+quasinormalform;
+quores;
+quores_integer;
+quores_longint;
+quores_monopoly;
+quores_monopoly_pre300402;
+raise_power_monopoly;
+random_adjacency_matrix;
+random_bar;
+random_bruch;
+random_bv;
+random_char_ff;
+random_ff;
+random_ff_given_q;
+random_generator_diag_subgroup;
+random_generator_sk_subgroup;
+random_generator_small_glkq_singersubgroup;
+random_gral;
+random_gr_given_c_d;
+random_integer;
+random_integervector;
+random_kranz;
+random_longint;
+random_monom;
+random_part_EXPONENT;
+random_partition_exponent;
+random_partition;
+random_permutation;
+random_polynom;
+random_reihe;
+random_subgroup_glk_grcd_2gen;
+random_subgroup_glk_grcd_cyclic;
+random_subgroup_glk_grcd_diagonal;
+random_subgroup_glk_grcd;
+random_subgroup_glk_grcd_smaller_k;
+random_subgroup_glk_grcd_stabilizer;
+random_subgroup_glkq;
+random_subgroup_glkq_1gen;
+random_subgroup_glkq_2gen;
+random_subgroup_glkq_3gen;
+random_subgroup_glkq_block;
+random_subgroup_glkq_cyclic;
+random_subgroup_glkq_diagonal;
+random_subgroup_glkq_perm_cyclic;
+random_subgroup_glkq_perm_diagonal;
+random_subgroup_glkq_perm;
+random_subgroup_glkq_prim;
+random_subgroup_glkq_smaller_k;
+random_subgroup_glkq_upper_cyclic;
+random_word;
+rank;
+rank_ff;
+rank_k_subset;
+rank_line;
+rank_permutation;
+rank_point;
+red_dia_perm;
+redf_cap;
+redf_cap_hoch;
+redf_cup;
+redf_cup_hoch;
+reduce_ff;
+reduce_inner_tensor_sc;
+reduce_nc;
+reduce_nc_kranz;
+reduceninpaar;
+reduce_symchar;
+reduce_symchar_tafel;
+reell_gram_schmidt;
+release_numbers;
+remove_hook;
+remove_mp_qnumber_fac;
+removepartij;
+remove_part_integer;
+remove_part_part;
+remove_vec_qnumber;
+remove_zero_terms;
+reorder_hall_littlewood;
+reorder_schur;
+reorder_vector;
+reorder_vector_apply;
+represent_hecke_element;
+reset_basis;
+reset_saving;
+reverse_bitvector;
+reverse_vector;
+rg_factorize;
+rg_order_ff;
+rg_quasi_normal_form;
+rh_test;
+ribbon_matrix;
+ribbon_partition;
+rightkey_wt;
+rindexword_sub;
+rindexword;
+rm_rindex;
+root_dimension;
+root_normalise_monopoly;
+root_represent_hecke_action;
+root_standardise_cold_tableaux_list;
+row_column_matrices;
+rowwordoftableaux;
+R_roftableaux;
+runtime;
+rz;
+rz_bar;
+rz_Dn;
+rz_lehmercode_bar;
+rz_lehmercode;
+rz_perm;
+sab;
+sab_input;
+S_a_rofword;
+save_cyclo_list;
+Sbar_nkq_maxgrad;
+s_b_i;
+s_bm_hi;
+s_bm_li;
+s_b_o;
+s_b_oi;
+s_b_u;
+s_b_ui;
+s_bv_li;
+scalarp;
+scalarproduct;
+scalarproduct_bar_schubert;
+scalarproduct_elmsym;
+scalarproduct_homsym;
+scalarproduct_homsym_monomial;
+scalarproduct_monomial;
+scalarproduct_nc;
+scalarproduct_powsym;
+scalarproduct_powsym_powsym;
+scalarproduct_schubert;
+scalarproduct_schur;
+scalarproduct_schur_schur;
+scalarproduct_symchar;
+scalarproduct_vector;
+scale_monopoly;
+scan_bar;
+scan_bitvector;
+scan_bruch;
+scan_cyclo;
+scan_elmsym;
+scan_exponentpartition;
+scan_fastpolynom;
+scan_ff;
+scan_galois;
+scan_gl_nc;
+scan_gral;
+scan_homsym;
+scan_integerbruch;
+scan_integer;
+scan_integermatrix;
+scan_integervector;
+scan;
+scan_kostka;
+scan_kranz;
+scan_laurent;
+scan_list;
+scan_longint;
+scan_matrix;
+scan_monom;
+scan_monomial;
+scan_monopoly;
+scan_nc_kranz;
+scanobjectkind;
+scan_partition;
+scan_parttableaux;
+scan_permutation;
+scan_permutation_cycle;
+scan_permvector;
+scan_polynom;
+scan_powsym;
+scan_printeingabe;
+scan_reihe;
+scan_reversepartition;
+scan_schubert;
+scan_schur;
+scan_skewpartition;
+scan_skewsymmetric_matrix;
+scan_skewtableaux;
+scan_sqrad;
+scan_symchar;
+scan_tableaux;
+scan_vector;
+scan_word;
+schensted_row_delete_step;
+schensted_row_insert_step;
+schen_word;
+schicht_print;
+schnitt_mat;
+schnitt_schur;
+schubertp;
+schur_ende;
+schur_part_skewschur;
+schur_powerproduct_schur_plet_mult;
+schur_power_schur_plet_mult;
+schur_schur_pletbis;
+schur_schur_plet;
+sdg;
+sd_i_wt;
+Sd_wt;
+select_coeff_polynom;
+select_coeff_reihe;
+select_column;
+select_column_tableaux;
+select_degree_reihe;
+select_i;
+select_row;
+select_row_tableaux;
+set_bm_ij;
+set_bv_i;
+set_cyclotomic_parameters;
+set_lo_nopoint;
+set_root_parameters;
+set_tex_polynom_parameter;
+setup_numbers;
+set_useful_monopolies;
+s_ff_c;
+s_ff_ci;
+s_ff_di;
+s_ff_ii;
+s_gr_k;
+s_gr_kn;
+s_gr_kni;
+s_gr_koor;
+s_gr_koori;
+s_gr_na;
+s_gr_nai;
+s_gr_s;
+s_gr_xkoori;
+s_gr_ykoori;
+signum;
+signum_permutation;
+s_i_i;
+singularp;
+Sinus_eingabe;
+s_i_wt;
+skewpartitionp;
+skewplane_plane;
+skip_comment;
+skip_integer;
+skip;
+s_ki_wt;
+s_kr_g;
+s_kr_gi;
+s_kr_gl;
+s_kr_gli;
+s_kr_i;
+s_kr_v;
+Sk_wt;
+s_lc_poly;
+s_l_n;
+s_l_s;
+s_m_h;
+s_m_hash;
+s_m_hi;
+s_m_ij;
+s_m_iji;
+s_m_l;
+s_m_li;
+s_mo_k;
+s_mo_ki;
+s_mo_s;
+s_mo_si;
+s_mo_sii;
+s_mo_sl;
+s_mo_sli;
+s_m_s;
+s_mz_po;
+s_mz_v;
+s_mz_vi;
+s_mz_vii;
+sn_character;
+s_n_d;
+s_n_dcd;
+s_n_dci;
+s_n_dcp;
+sn_dimension;
+s_ni_wt;
+S_nkq_maxgrad;
+s_n_s;
+sn_tableaux;
+sn_weintraube;
+Sn_wt;
+so_character;
+so_dimension;
+s_o_k;
+SYM_sort;
+sort_rows_tableaux_apply;
+sort_vector;
+s_o_s;
+so_tableaux;
+s_pa_hash;
+s_pa_i;
+s_pa_ii;
+s_pa_k;
+s_pa_l;
+s_pa_li;
+spaltenanfang;
+spaltenende;
+spalten_summe;
+s_pa_s;
+sp_br;
+sp_character;
+sp_dimension;
+specht_dg;
+specht_irred_characteristik;
+specht_m_part_sc;
+specht_poly;
+specht_powersum;
+special_eq;
+special_mult_apply_homsym_homsym;
+speicher_anfang;
+speicher_ende;
+s_p_i;
+s_p_ii;
+spin_tableaux_character;
+s_p_k;
+s_p_l;
+s_p_li;
+split_hashtable;
+split;
+splitpart;
+s_po_k;
+s_po_ki;
+s_po_n;
+s_po_s;
+s_po_si;
+s_po_sii;
+s_po_sl;
+s_po_sli;
+sprint_ff;
+sprint_integer;
+sprint_integervector;
+sprint_longint;
+sprint_partition;
+sprint_permutation;
+sprint_skewpartition;
+sprint;
+sprint_vector;
+s_p_s;
+sp_tableaux;
+square_apply;
+square_apply_integer;
+square_apply_longint;
+square_free_part_0;
+square_free_part;
+squareroot;
+squareroot_bruch;
+squareroot_integer;
+squareroot_longint;
+S_rofword;
+sscan_bar;
+sscan_bitvector;
+sscan_elmsym;
+sscan_homsym;
+sscan_integer;
+sscan_integervector;
+sscan_longint;
+sscan_partition;
+sscan_permutation;
+sscan_permvector;
+sscan_reversepartition;
+sscan_schur;
+sscan;
+sscan_word;
+s_sc_d;
+s_sc_di;
+s_sch_k;
+s_sch_ki;
+s_sch_n;
+s_sch_s;
+s_sch_si;
+s_sch_sii;
+s_sch_sl;
+s_sch_sli;
+s_sc_p;
+s_sc_pi;
+s_sc_pli;
+s_sc_w;
+s_sc_wi;
+s_sc_wii;
+s_sc_wli;
+s_s_k;
+s_s_ki;
+s_s_n;
+s_spa_g;
+s_spa_gi;
+s_spa_gii;
+s_spa_gli;
+s_spa_k;
+s_spa_ki;
+s_spa_kii;
+s_spa_kli;
+s_s_s;
+s_s_si;
+s_s_sii;
+s_s_sl;
+s_s_sli;
+standardise_cold_tableaux_list;
+standardise_cyclo;
+standardp;
+starpart;
+starting_bar_schubert;
+start_longint;
+starttableaux;
+std_perm;
+s_t_h;
+s_t_hi;
+s_t_ij;
+s_t_iji;
+stirling_first_tafel;
+stirling_numbers_second_kind_vector;
+stirling_second_number_kostka;
+stirling_second_number;
+stirling_second_number_tafel;
+stirling_second_tafel;
+s_t_l;
+s_t_li;
+store_result_0;
+store_result_1;
+store_result_2;
+store_result_3;
+store_result_4;
+store_result_5;
+strictp;
+strict_to_odd_part;
+strong_check_barp;
+strong_check_permutationp;
+strong_generators;
+s_t_s;
+s_t_u;
+s_t_ug;
+s_t_ugii;
+s_t_ugli;
+s_t_ui;
+s_t_uii;
+s_t_uk;
+s_t_ukii;
+s_t_ukli;
+s_t_ul;
+s_t_uli;
+s_t_us;
+sub;
+sub_apply;
+sub_comp_bv;
+sub_comp_part;
+sub_default;
+sub_part_part;
+substitute_one_matrix;
+substitute_one_monopoly;
+SYM_sum;
+sum_integervector;
+sum_matrix;
+sum_vector;
+sup_bitvector_apply;
+sup_bitvector;
+s_v_i;
+s_v_ii;
+s_v_l;
+s_v_li;
+s_v_s;
+swap;
+s_w_i;
+s_w_ii;
+s_w_l;
+s_w_li;
+s_w_s;
+S_wt;
+s_x_nu_to_ypolynom;
+symchar_hoch_n;
+SYMMETRICA_bricknumber;
+SYMMETRICA_EH;
+SYMMETRICA_EM;
+SYMMETRICA_EP;
+SYMMETRICA_ES;
+SYMMETRICA_HE;
+SYMMETRICA_HM;
+SYMMETRICA_HP;
+SYMMETRICA_HS;
+SYMMETRICA_ME;
+SYMMETRICA_MH;
+SYMMETRICA_MP;
+SYMMETRICA_MS;
+SYMMETRICA_PE;
+SYMMETRICA_PH;
+SYMMETRICA_PM;
+SYMMETRICA_PS;
+SYMMETRICA_SE;
+SYMMETRICA_SH;
+SYMMETRICA_SM;
+SYMMETRICA_SP;
+symmetricp;
+symmetricp_i;
+symmetricp_matrix;
+t_2SCHUBERT_POLYNOM;
+tab_anfang;
+tab_ende;
+tab_funk;
+tableaux_character;
+tableauxp;
+table_of_gauss_numbers;
+table_of_Rbar_nkq;
+table_of_R_nkq;
+table_of_Sbar_nkq;
+table_of_S_nkq;
+table_of_Tbar_nkq;
+table_of_T_nkq;
+table_of_W_nkq;
+tab_wt;
+t_augpart_part;
+t_BAR_BARCYCLE;
+t_BARCYCLE_BAR;
+t_bar_doubleperm;
+Tbar_nkq_file;
+Tbar_nkq_maxgrad;
+t_BINTREE_ELMSYM;
+t_BINTREE_ELMSYM_apply;
+t_BINTREE_GRAL;
+t_BINTREE_HOMSYM;
+t_BINTREE_HOMSYM_apply;
+t_BINTREE_LIST;
+t_BINTREE_MONOMIAL;
+t_BINTREE_MONOMIAL_apply;
+t_BINTREE_POLYNOM;
+t_BINTREE_POLYNOM_apply;
+t_BINTREE_POWSYM;
+t_BINTREE_POWSYM_apply;
+t_BINTREE_SCHUBERT;
+t_BINTREE_SCHUR;
+t_BINTREE_SCHUR_apply;
+t_BINTREE_VECTOR;
+t_BIT_VECTOR;
+t_BITVECTOR_INTVECTOR;
+t_BRUCH_LAURENT;
+teh_elmsym__faktor;
+teh_ende;
+teh___faktor;
+teh_hashtable__faktor;
+teh_integer__faktor;
+teh_partition__faktor;
+teh_partition__faktor_pre290102;
+t_ELMSYM_ELMSYM;
+t_ELMSYM_HASHTABLE;
+t_ELMSYM_HOMSYM;
+t_ELMSYM_MONOMIAL;
+t_ELMSYM_POWSYM;
+t_ELMSYM_SCHUR;
+t_ELMSYM_SCHUR_pre041201;
+tem_ende;
+tem_integer__faktor;
+tem_partition__faktor;
+tep_elmsym__faktor;
+tep_ende;
+tep___faktor;
+tep_hashtable__faktor;
+tep_integer__faktor;
+tep_partition__faktor;
+tep_partition__faktor_pre040202;
+tes_elmsym__faktor;
+tes___faktor;
+tes_hashtable__faktor;
+tes_integer__faktor;
+tes_partition__faktor;
+test_bintree;
+test_brc;
+test_bruch;
+test_callocobject;
+test_dcp;
+test_integer;
+test_kostka;
+test_list;
+test_longint;
+test_matrix;
+test_mdg;
+test_ndg;
+test_number;
+test_object;
+test_perm;
+test_plet;
+test_poly;
+test_schubert;
+test_schur;
+test_symchar;
+test_word;
+tex_2schubert_monom_summe;
+tex_bruch;
+tex_cyclo;
+tex_dia_wt;
+tex_hall_littlewood;
+tex_hecke_monopoly;
+tex_integer;
+tex_kostka;
+tex_lc;
+tex_list;
+tex_longint;
+tex_matrix_co;
+tex_matrix;
+tex_monom;
+tex_monom_plus;
+tex_monopoly;
+tex;
+tex_partition;
+tex_permutation;
+tex_polynom;
+t_EXPONENT_VECTOR;
+t_EXPONENT_VECTOR_apply;
+tex_rz;
+tex_schubert;
+tex_schur;
+tex_sqrad;
+tex_symchar;
+tex_tableaux;
+tex_vector;
+tex_weintraube;
+t_forall_monomials_in_a;
+t_FROBENIUS_VECTOR;
+t_galois_polynom;
+t_HASHTABLE_ELMSYM;
+t_HASHTABLE_ELMSYM_apply;
+t_HASHTABLE_HOMSYM;
+t_HASHTABLE_HOMSYM_apply;
+t_HASHTABLE_MONOMIAL;
+t_HASHTABLE_MONOMIAL_apply;
+t_HASHTABLE_POLYNOM;
+t_HASHTABLE_POLYNOM_apply;
+t_HASHTABLE_POWSYM;
+t_HASHTABLE_POWSYM_apply;
+t_HASHTABLE_SCHUR;
+t_HASHTABLE_SCHUR_apply;
+t_HASHTABLE_VECTOR;
+the_integer__faktor;
+thm_ende;
+thm_integer__faktor;
+thm_partition__faktor;
+t_HOMSYM_ELMSYM;
+t_HOMSYM_HASHTABLE;
+t_HOMSYM_HOMSYM;
+t_HOMSYM_MONOMIAL;
+t_HOMSYM_POWSYM;
+t_HOMSYM_SCHUR;
+thp_ende;
+thp___faktor;
+thp_homsym__faktor;
+thp_integer__faktor;
+thp_partition__faktor;
+thp_partition__faktor_pre300102;
+tidy;
+t_INTEGER_FF;
+t_INTEGER_LAURENT;
+t_int_longint;
+t_INTVECTOR_BITVECTOR;
+t_INTVECTOR_UCHAR;
+t_j_wt;
+t_LAURENT_OBJ;
+t_LIST_POLYNOM;
+t_LIST_VECTOR;
+t_longint_int;
+t_loop_partition;
+tl_set_index_inc;
+tl_set_max_numb;
+tl_set_prime;
+t_MA_MONOPOLY_MA_POLYNOM;
+tme_ende;
+tme___faktor;
+tme_hashtable__faktor;
+tme_integer__faktor;
+tme_monomial__faktor;
+tme_partition__faktor;
+tmh_ende;
+tmh___faktor;
+tmh_integer__faktor;
+tmh_monomial__faktor;
+tmh_partition__faktor;
+t_MONOMIAL_ELMSYM;
+t_MONOMIAL_HASHTABLE;
+t_MONOMIAL_HOMSYM;
+t_MONOMIAL_MONOMIAL;
+t_MONOMIAL_POWSYM;
+t_MONOMIAL_SCHUR;
+t_MONOPOLY_LAURENT;
+t_MONOPOLY_POLYNOM;
+tmp___faktor;
+tmp_integer__faktor;
+tmp_int__faktor;
+tmp_monomial__faktor;
+tmp_partition__faktor;
+T_nkq_file;
+T_nkq_maxgrad;
+T_nkq_maxgrad_erweitert;
+t_OBJ_LAURENT;
+t_PARTITION_AUGPART;
+tpe___faktor;
+tpe_hashtable__faktor;
+tpe_integer__faktor;
+tpe_partition__faktor;
+tpe_partition__faktor_pre300102;
+tpe_powsym__faktor;
+t_PERMUTATION_SCHUBERT;
+tph___faktor;
+tph_hashtable__faktor;
+tph_integer__faktor;
+tph_partition__faktor;
+tph_partition__faktor_pre300102;
+tph_powsym__faktor;
+tpm___faktor;
+tpm_hashtable__faktor;
+tpm_integer__faktor;
+tpm_partition__faktor;
+tpm_powsym__faktor;
+t_POLYNOM_ELMSYM;
+t_polynom_galois;
+t_POLYNOM_LAURENT;
+t_POLYNOM_MONOMIAL;
+t_POLYNOM_MONOPOLY;
+t_POLYNOM_POWER;
+t_POLYNOM_SCHUBERT;
+t_POLYNOM_SCHUR;
+t_POLYNOM_T;
+t_POWSYM_ELMSYM;
+t_POWSYM_HASHTABLE;
+t_POWSYM_HOMSYM;
+t_POWSYM_MONOMIAL;
+t_POWSYM_POWSYM;
+t_POWSYM_SCHUR;
+t_productexponent;
+tps___faktor;
+trace;
+trace_matrix;
+trafo_check;
+trans2formlist;
+transform_apply_list;
+transformlist;
+trans_index_monopoly_cyclo;
+transpose;
+transpose_matrix;
+transpose_second_matrix;
+t_REIHE_POLYNOM;
+t_SCHUBERT_POLYNOM;
+t_SCHUBERT_SCHUR;
+t_SCHUR_ELMSYM;
+t_SCHUR_HASHTABLE;
+t_SCHUR_HOMSYM;
+t_schur_jacobi_trudi;
+t_SCHUR_MONOMIAL;
+t_SCHUR_MONOMIAL_pre211101;
+t_schur_naegelsbach;
+t_SCHUR_POWSYM;
+t_SCHUR_SCHUR;
+t_SCHUR_SYMCHAR;
+tse___faktor;
+tse___faktor_slow;
+tse_integer__faktor;
+tse_partition__faktor;
+tse_partition__faktor_pre040202;
+tse_schur__faktor;
+tsh_eval_jt;
+tsh___faktor;
+tsh_hashtable__faktor;
+tsh_integer__faktor;
+tsh_jt;
+tsh_partition__faktor;
+tsh_partition__faktor_pre240102;
+tsh_partition__faktor_pre310102;
+tsh_schur__faktor;
+tsm___faktor;
+tsm_integer__faktor;
+tsm_partition__faktor;
+tsm_schur__faktor;
+tsp___faktor;
+tsp_integer__faktor;
+t_splitpart;
+tsp_partition__faktor;
+tsp_schur__faktor;
+t_UCHAR_INTVECTOR;
+t_VECTOR_BIT;
+t_VECTOR_BITREC;
+t_VECTOR_EXPONENT;
+t_VECTOR_FROB;
+t_VECTOR_FROBENIUS;
+t_VECTOR_LIST;
+t_VECTOR_POLYNOM;
+t_VECTOR_ZYKEL;
+t_vperm_zperm;
+T_weintraube;
+twoword_knuth;
+twoword_matrix;
+txx_null__faktor;
+typusorder;
+t_zperm_vperm;
+t_ZYKEL_VECTOR;
+UD_permutation;
+umriss_tableaux;
+unimodalp;
+unitp_galois;
+unrank_degree_permutation;
+unrank_given_q_ff;
+unrank_given_qk_point;
+unrank_k_subset;
+unrank_permutation;
+unrank_subset;
+unset_bm_ij;
+unset_bv_i;
+usersort_vector;
+vander;
+variety_poly;
+vec_anfang;
+vec_ende;
+vec_mat_mult;
+vectorofzerodivisors_galois;
+vectorp;
+vertikal_sum;
+vexillaryp;
+vexillaryp_permutation;
+vminus_hecke;
+vminus_tabloid;
+vminus;
+vorgaenger_bruhat;
+vorgaenger_bruhat_strong;
+vorgaenger_bruhat_weak;
+vorgaenger_young;
+vorgaenger_young_skewpartition;
+weight;
+weight_augpart;
+weight_partition;
+weight_skewpartition;
+weight_tableaux;
+weight_vector;
+weintraube_monom;
+weintrauben_polynom;
+weintraube_vector;
+werte_Polynom_aus;
+wert;
+which_part;
+W_nkq_maxgrad;
+wordoftableaux;
+word_schen;
+word_tableaux;
+word_wt;
+wrong_type_oneparameter;
+wrong_type_twoparameter;
+wt_liste_gewicht;
+wt_word;
+ym_min;
+young_alt_scalar_tafel;
+young_character;
+young_ideal;
+youngp;
+young_polynom;
+young_scalar_tafel;
+young_tafel;
+zeilenanfang;
+zeilenende;
+zeilen_summe;
+zentralprim;
+zerlege_nk_codes;
+zykeldarstellung_matrix;
+zykeldarstellung_matrix_red;
+zykelind_aff1Zn;
+zykelind_affkq;
+zykelind_affkzn;
+zykelind_An;
+zykelind_arb;
+zykelind_centralizer;
+zykelind_Cn;
+zykelind_cube;
+zykelind_cube_edges;
+zykelind_cube_edges_extended;
+zykelind_cube_extended;
+zykelind_cube_faces;
+zykelind_cube_faces_extended;
+zykelind_cube_vertices;
+zykelind_cube_vertices_extended;
+zykelind_dec;
+zykelind_dec_apply;
+zykelind_dir_prod;
+zykelind_dir_prod_apply;
+zykelind_dir_summ;
+zykelind_dir_summ_apply;
+zykelind_Dn;
+zykelind_dodecahedron;
+zykelind_dodecahedron_edges;
+zykelind_dodecahedron_edges_extended;
+zykelind_dodecahedron_extended;
+zykelind_dodecahedron_faces;
+zykelind_dodecahedron_faces_extended;
+zykelind_dodecahedron_vertices;
+zykelind_dodecahedron_vertices_extended;
+zykelind_exponentiation;
+zykelind_glkq;
+zykelind_glkzn;
+zykelind_hoch_dir_prod;
+zykelind_hoch_dir_summ;
+zykelind_inc;
+zykelind_kranz;
+zykelind_on_2sets;
+zykelind_on_ksubsets;
+zykelind_on_ktuples;
+zykelind_on_ktuples_injective;
+zykelind_on_pairs;
+zykelind_on_pairs_disjunkt;
+zykelind_on_pairs_oriented;
+zykelind_on_pairs_reduced;
+zykelind_on_power_set;
+zykelind_pglkq;
+zykelind_plethysm;
+zykelind_Sn;
+zykelind_stabilizer_part;
+zykelind_superp_lin_dir_graphs;
+zykelind_test;
+zykelind_tetraeder;
+zykelind_tetraeder_edges;
+zykelind_tetraeder_edges_extended;
+zykelind_tetraeder_extended;
+zykelind_tetraeder_faces;
+zykelind_tetraeder_faces_extended;
+zykelind_tetraeder_vertices;
+zykelind_tetraeder_vertices_extended;
+zykeltyp;
+zykeltyp_hoch_n;
+zykeltyp_permutation;
+zykeltyp_pi_hoch;
+partition_speichersize;
+partition_speicherindex;
+mem_counter_part;
+partition_speicher;
+monom_speicherindex;
+mem_counter_monom;
+monom_speichersize;
+monom_speicher;
+loc_index;
+loc_size;
+loc_counter;
+loc_speicher;
+longint_speicherindex;
+mem_counter_loc;
+longint_speichersize;
+longint_speicher;
+
+local: *;
+};
